libqtile/notify.py

```diff
@@ -29,29 +30,24 @@
 from libqtile.log_utils import logger
 
 try:
-    import dbus
-    from dbus import service
-    from dbus.mainloop.glib import DBusGMainLoop
-    DBusGMainLoop(set_as_default=True)
-    has_dbus = True
+    from dbus_next.aio import MessageBus  # noqa: I001
+    from dbus_next.service import method, signal, ServiceInterface  # noqa: I001
+    has_dbus = True  # noqa: I005S
 except ImportError:
```

Comment:
You can probably just dispense with this. dbus_next looks lightweight enough that we can just require it.